src/us/kbase/typedobj/core/TypeDefName.java

```diff
@@ -8,8 +8,8 @@
 
 public class TypeDefName {
 	
-	private final static Pattern INVALID_TYPE_NAMES = 
-			Pattern.compile("[^\\w]");
+	// In java, unlike, say, python 3, this is a-zA-Z0-9_ only, no unicode chars
```

Comment:
wtf python 3?!

Reply:
I wanted to say "unlike, say, lesser languages like python 3" but there are just too many people around that can't emotionally handle those kinds of foundational truths